src/com/robibp/trening/UnosKlinca.java

```diff
@@ -0,0 +1,38 @@
+package com.robibp.trening;
+
+import java.util.Scanner;
+
+class UnosKlinca {
+
+    private boolean samoSlova(String ime) {
```

Comment:
Uvijek poredaj metode po visibility-u. To znaci ovako: `public`, `protected`, `package-private`, `private`